app/controllers/users_controller.rb

```diff
@@ -8,7 +8,14 @@ def index
     @users = User.paginate(page: params[:page])
   end
 
-  def show; end
+  def show
+    @user = User.find_by id: params[:id]
+    @microposts = @user.microposts.paginate(page: params[:page])
+    if @user.nil?
```

Comment:
it should check `if @user.nil?`on line 13, before `@microposts = @user.microposts.paginate(page: params[:page])`
because if @user is nil. `@user.microposts.paginate(page: params[:page])` will raise errors